ci/bump_version.sh

```diff
@@ -1,12 +1,32 @@
 #!/bin/bash
 
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+BLUE='\033[0;34m'
+MAGENTA='\033[0;35m'
+CYAN='\033[0;36m'
+GRAY='\033[0;37m'
+
+# Reset
+NC='\033[0m'
+
 if [ $# != 1 ]
 then
      echo "Please provide version as the first argument"
```

Comment:
Maybe you want to use your colors also for this message?

Reply:
I think this one is better off without any colours.